workspaces/sample-default-workspace/client-extensions/easy/src/main/resources/application.properties

```diff
@@ -4,4 +4,5 @@ logging.level.org.springframework.security=DEBUG
 logging.level.org.springframework.web=DEBUG
 liferay.oauth.application.external.reference.code=easy-oauth-application-user-agent
 liferay.portal.url=http://localhost:8080
+dxp.domains=localhost
```

Comment:
Hey Ray can you sort this?

Reply:
hey, this is a superfluous property so I will remove it. thx